.eslintrc.json

```diff
@@ -5,7 +5,13 @@
     "project": ["./tsconfig.json"]
   },
   "env": { "es6": true },
-  "ignorePatterns": ["node_modules", "build", "coverage", "proto"],
+  "ignorePatterns": [
+    "node_modules",
+    "build",
+    "coverage",
+    "proto",
+    "**/webpack.config.js"
```

Comment:
it doesn't seem to be needed anymore as `webpack.config.js` looks good now from lint perspective